DHCP graph for Vlan

stveit opened the pull request:
Fixes #2373 

Graph dhcp statistics on vlan page

stveit committed:
Add metric path for dhcp stats

stveit committed:
Add DHCP statistics graph to vlan page

stveit committed:
Dont draw graph if no DHCP stats exist

stveit committed:
Represent vlan using object string

stveit committed:
Fix dhcp graph not showing if no new events

if there havent been added any new datapoints in gaphite recently  (5 or 10 minutes
 it seems) then the previous function would not work, even if you changed
start to 1year instead of 5 min

stveit committed:
Use better suited function for checking if metric exists

stveit committed:
Rename variables

maybe shouldnt use max as a variable name

lunkwill42 reviewed (changes requested):
This looks good so far, but the failing tests need to be addresses, as mentioned in my inline comments.

Also (and I realize this is on me, not having formally defined the metric path "standard" yet), the metric paths will not necessarilly conform to `nav\.dhcp\.vlan\d+\.`. As the original spec says in #2369:

> The network names can also be something like vlan1100_some_description, or some_description_vlan1100, but this should still match as VLAN 1100 in NAV.
>
> An extra level in the metric path for location may also be needed. This could in reality be any prefix configured into the integration script, something like:

VLAN numbers can be reused, even though they are separate broadcast domains. This typically happens on separate locations in a WAN, though - so be prepared that the path pattern may become a bit more complex...

stveit committed:
Show alert if graphite is unreachable

stveit committed:
Check if graph exists using new variable

stveit commented on python/nav/web/templates/info/vlan/vlandetails.html:
```diff
@@ -154,4 +157,15 @@ <h2>Prefix graphs</h2>
+  <h2>DHCP graphs</h2>
+
+  <ul class="small-block-grid-1 large-block-grid-2">
+    {% if vlan.has_dhcp_stats %}
```
graphite error fixed, so i guess this is on hold until the metric path is defined

stveit committed:
Dont render graph if no statistics exist

this check was existed, then was removed in a previous commit
when checking for connectiong errors to graphite, and now
im bringing it back

stveit committed:
Check for graphite error with has_dhcp_stats
